dotnet/src/Experimental/Process.Abstractions/KernelProcessEvent.cs

```diff
@@ -3,19 +3,19 @@
 namespace Microsoft.SemanticKernel;
 
 /// <summary>
-/// An class representing an event that can be emitted from a <see cref="KernelProcessStep"/>. This type is convertible to and from CloudEvents.
+/// A class representing an event that can be emitted from a <see cref="KernelProcessStep"/>. This type is convertible to and from CloudEvents.
 /// </summary>
-public sealed record KernelProcessEvent
+public record KernelProcessEvent
```

Comment:
can this still be sealed?

Reply:
Yes, left over from playing with generics.